app/controllers/static_pages_controller.rb

```diff
@@ -16,7 +16,11 @@ def copyright
   def home
     flash.keep # keep notices and errors through to the redirects below
 
-    if signed_in?
+    if signed_in? && Settings::Db.store.newflow_feature_flag
```

Comment:
This reads a little funky to me, like hard to see that you have half of the `if` that activates only for the new flow.  Can you nest `if` statements here to not check the same thing twice?

```ruby
if Settings::Db.store.newflow_feature_flag
  signed_in? ? redirect_to(profile_newflow_path) : newflow_authenticate_user!
else
  signed_in? ? redirect_to(profile_path) : authenticate_user! 
end
```

Reply:
Fixed. Thanks for the refactor, I do like it much better.